airbyte-integrations/connectors/source-airtable/oauth2.patch.json

```diff
@@ -0,0 +1,8 @@
+{
+    "provider": "airtable",
+    "authUrlTemplate": "https://airtable.com/oauth2/v1/authorize?client_id={{#urlencode}}{{{ client_id }}}{{/urlencode}}&redirect_uri={{#urlencode}}{{{ redirect_uri }}}{{/urlencode}}&response_type=code&state={{#urlencode}}{{{ state }}}{{/urlencode}}&scope=data.records:read%20data.recordComments:read%20schema.bases:read    ",
+    "accessTokenUrlTemplate": "https://airtable.com/oauth2/v1/token",
+    "accessTokenBody": "grant_type=authorization_code&client_id={{#urlencode}}{{{ client_id }}}{{/urlencode}}&client_secret={{#urlencode}}{{{ client_secret }}}{{/urlencode}}&redirect_uri={{#urlencode}}{{{ redirect_uri }}}{{/urlencode}}&code={{#urlencode}}{{{ code }}}{{/urlencode}}",
+    "accessTokenHeaders": {"content-type": "application/x-www-form-urlencoded"},
+    "accessTokenResponseMap": {"refresh_token": "/refresh_token"}
```

Comment:
The response includes more fields. If you look at `spec.json` of this connector, we see three fields that the connector expects to receive from the endpoint: `access_token`, `refresh_token` and `token_expires_in`. In the [AirTable docs](https://airtable.com/developers/web/api/oauth-reference#token-creation-request) you can find the original names of these fields. In this case, we would have:

```
{
  "refresh_token": "/refresh_token",
  "access_token": "/access_token",
  "token_expires_in": "/expires_in"
}
```

Reply:
@mdibaiee when I searched spec.json, I don't see `token_expires_in`, are you referring to `token_expiry_date`, or am I not looking at the right place?

Reply:
@jman0119 yes I think that's a typo from me, `token_expiry_date` 👍🏽